collections/_article/progress-report-state-of-particles.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "State of particles and future updates"
+excerpt: "An update on the current state of the particle system in Godot, and some hints about what the future holds!"
+categories: ["progress-report"]
+author: Ilaria Cislaghi
+image: /storage/blog/covers/progress-report-state-of-particles.webp
+date: 2023-10-17 14:00:00
+---
+
+_TL;DR: In 4.2 GPUParticles were refactored and lots of new features were added. Due to high maintenance cost and bug surface, CPUParticles will keep the current feature set._
+
+-----
+
+Godot 4 offers a great opportunity to make some long-requested and much needed changes in many areas, and one of those areas is particle nodes and resources. Before 4.0 some settings used to be a bit confusing and clunky to work with. Now a lot of things have changed: parameters are set with a range using min and max boundaries instead of value and random deviation; there's a particle collision system; there's a turbulence system, there are sub-emitters, and much more.
+
+All around, particles already received a lot of love during the development of Godot 4. However, a number of key features were still missing as of 4.1. Below I go into detail about additions and improvements made to the particle system in the last few months.
+
+<video autoplay loop muted playsinline title="A countdown with particle effects">
+  <source src="/storage/blog/state-of-particles/final-countdown.mp4" type="video/mp4">
+</video>
+
+## Artistic control
+
+Godot's existing particle system made an assumption that people would use it for more physically accurate simulations, such which are suited for more realistic games. But in practice a vast majority of projects in the Godot ecosystem choose a stylized art direction.
+
+This creates a big pain point when using current particles. In order to obtain an exaggerated motion and snappy timings in visual effects, acceleration-based approach just doesn't cut it. It ends up really clunky, hard to use, and sometimes achieving certain effects is just not possible.
+
+To address these issues, and more, the `GPUParticles` system has been refactored and expanded upon. Particles can now support not just realistic types of effects, but also more stylized ones with more exaggerated motion. Additionally, a couple of long-requested quality of life features was added.
+
+## The new `GPUParticles`
+
+**NOTE:** These changes should be 100% backwards compatible. If particles look different after updating, please [file a bug report](https://github.com/godotengine/godot/issues/new/choose).
+
+_[Pioneering of this work](https://github.com/godotengine/godot/pull/78851) was kindly sponsored by [PahdoLabs](https://www.pahdolabs.com/)._
```

Comment:
It might be a bit confusing that this links to an unmerged PR. Maybe it should link both 78851 as the one pioneering that work, and then 79527? E.g.:
```suggestion
_[Pioneering of this work](https://github.com/godotengine/godot/pull/78851) was kindly sponsored by [Pahdo Labs](https://www.pahdolabs.com/)._ After further iteration, this was eventually merged in [GH-79527](https://github.com/godotengine/godot/pull/79527).
```
(Also space in "Pahdo Labs", at least judging by their website.)